src/storage-client/src/controller.rs

```diff
@@ -259,10 +259,13 @@ pub trait StorageController: Debug + Send {
     /// The method returns a oneshot that can be awaited to indicate completion of the write.
     /// The method may return an error, indicating an immediately visible error, and also the
     /// oneshot may return an error if one is encountered during the write.
-    // TODO(petrosagg): switch upper to `Antichain<Timestamp>`
```

Comment:
I think this work is uncontroversial, but LMK if it turns out to be otherwise. we need it so we can pass in `[]`